src/components/ButtonPanel.js

```diff
@@ -1,42 +1,51 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-const ButtonPanel = () => (
-  <div className="flex flex-col btnPanel">
-    <div className="rowPanel">
-      <Button color="#d3d3d3" buttonName="AC" />
-      <Button color="#d3d3d3" buttonName="+/-" />
-      <Button color="#d3d3d3" buttonName="%" />
-      <Button buttonName="÷" />
-    </div>
+const ButtonPanel = props => {
+  const handleClickBP = buttonName => props.clickHandler(buttonName);
 
-    <div className="rowPanel">
-      <Button color="#d3d3d3" buttonName="7" />
-      <Button color="#d3d3d3" buttonName="8" />
-      <Button color="#d3d3d3" buttonName="9" />
-      <Button buttonName="X" />
-    </div>
+  return (
+    <div className="flex flex-col btnPanel">
+      <div className="rowPanel">
+        <Button color="#d3d3d3" buttonName="AC" clickHandler={handleClickBP} />
```

Comment:
- [ ] [OPTIONAL] If you save your colors into variables at a constants file or similar you could have a more readable code:
`<Button color={orange} buttonName="AC" c...`